hoge.rb

```diff
@@ -0,0 +1,2 @@
+a =  aa
+%w[1 'a',]
```

Comment:
__[development][rubocop]__
* Within `%w`/`%W`, quotes and ',' are unnecessary and may be unwanted in the resulting strings.

<!-- Comment by [SideCI](https://sideci.com) -->